doc/man/scons.xml

```diff
@@ -93,72 +93,67 @@
+rules exist for building many common software components
+(such as executable programs, object files, libraries),
+so that for most software projects,
+only the target and input files need be specified.</para>
 
-<para>By default,
-<command>scons</command>
+<para>
+When invoked, <command>scons</command>
 searches for a file named
-<emphasis>SConstruct</emphasis>,
-<emphasis>Sconstruct</emphasis>,
-<emphasis>sconstruct</emphasis>,
-<emphasis>SConstruct.py</emphasis>
-<emphasis>Sconstruct.py</emphasis>
-or
-<emphasis>sconstruct.py</emphasis>
-(in that order) in the current directory and reads its
-configuration from the first file found.
+&SConstruct;
+(going on to check alternative names
+&Sconstruct;, &sconstruct;, &SConstruct.py; &Sconstruct.py;
+and &sconstruct.py;
+in that order) in the current directory and reads its
+configuration from that file.
 An alternate file name may be
 specified via the
 <option>-f</option>
-option.</para>
-
-<para>The
-<emphasis>SConstruct</emphasis>
+option.
+The &SConstruct;
 file can specify subsidiary
 configuration files using the
-<emphasis role="bold">SConscript</emphasis>()
-function.
+&SConscriptFunc; function.
 By convention,
 these subsidiary files are named
-<emphasis>SConscript</emphasis>,
+&SConscript;,
 although any name may be used.
-(Because of this naming convention,
-the term "SConscript files"
-is sometimes used to refer
-generically to all
-<command>scons</command>
-configuration files,
-regardless of actual file name.)</para>
-
-<para>The configuration files
-specify the target files to be built, and
-(optionally) the rules to build those targets.  Reasonable default
-rules exist for building common software components (executable
-programs, object files, libraries), so that for most software
-projects, only the target and input files need be specified.</para>
-
-<para>Before reading the
-<emphasis>SConstruct</emphasis>
-file,
+As a result of this naming convention,
+the term <firstterm>SConscript files</firstterm>
+is often used to refer
+generically to the complete set of
+configuration files for a project,
+including the &SConstruct; file,
+regardless of the actual file names or number of files.</para>
+
+<para>Before reading the SConscript files,
 <command>scons</command>
 looks for a directory named
-<emphasis>site_scons</emphasis>
-in various system directories (see below) and the directory containing the
-<emphasis>SConstruct</emphasis>
-file; for each of those dirs which exists,
-<emphasis>site_scons</emphasis>
-is prepended to sys.path,
-the file
-<emphasis>site_scons/site_init.py</emphasis>,
-is evaluated if it exists,
+<filename>site_scons</filename>
+in various system directories and the directory containing the
+&SConstruct; file: each such directory which exists
+is prepended to <varname>sys.path</varname>;
+the file <filename>site_init.py</filename>
+is evaluated if it exists in that directory;
 and the directory
-<emphasis>site_scons/site_tools</emphasis>
```

Comment:
it should be site_scons/site_tools not just site_tools right?

Reply:
well it does say "in that directory" (the sentence is already about site_scons), but I'll think about if there's another way to word it... it's awkward language because there are three related things that happen and however you put it it seems to just run on...

Reply:
I updated that section.